src/main/webapp/app/exam/manage/exam-management.route.ts

```diff
@@ -299,6 +303,42 @@ export const examManagementRoute: Routes = [
         },
         canActivate: [UserRouteAccessService],
     },
+    {
+        path: ':examId/student-exams/:studentExamId/summary/exercises/:exerciseId/repository/:participationId',
+        component: RepositoryViewComponent,
+        data: {
+            authorities: [Authority.USER],
```

Comment:
This is on the management route. Is it intended that this is accessible by users or should this be TA and higher?

Reply:
Yes thank you :)